CMakeLists.txt

```diff
@@ -260,7 +260,9 @@ if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
 endif()
 
 if (NOT CMAKE_BUILD_TYPE)
-    set( CMAKE_BUILD_TYPE Release)
+   set( CMAKE_BUILD_TYPE Release)
+elseif(CMAKE_BUILD_TYPE STREQUAL "PureDebug")
```

Comment:
I'm not sure why we need to do any of this. CMake has a list of supported build types. I've never heard of PureDebug, and I don't know why anyone would try to enter it.

> Possible values are empty, Debug, Release, RelWithDebInfo and MinSizeRel.

From: https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html